recipes/wget/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+./configure --prefix=${PREFIX}        \
+            --enable-opie             \
+            --enable-digest           \
+            --enable-ntlm             \
+            --enable-debug            \
+            --with-ssl=openssl        \
+            --with-openssl=${PREFIX}  \
+            --with-zlib=${PREFIX}     \
+            --with-metalink           \
+            --with-cares              \
+            --with-libpsl
+make
+make install
```

Comment:
Is there a `make check` we can run?

Reply:
Good point added. It seems to catch the same problem that I wanted to find with my test case. Apparently, SSL does not work. The same holds for the wget from the default channel.

Reply:
I must admit that I have no idea what could be wrong. Any help appreciated.

Reply:
@johanneskoester I think the tests are failing because perl is missing. So not sure we can keep `make test` or at least we would need to add perl to the build-env?

Reply:
When we disable `make check`, my own test for ssl in the meta.yaml fails as well. So this is indeed at least also the SSL bug that has been seen in the default channel.